internal/participate/use-cases/nfts.md

```diff
@@ -1,27 +1,41 @@
 ---
-title: NFTs
-description: My document description from SEO
+title: Non-fungile Token (NFTs)
+description: This article explains the different types of NFTs on IOTA and how you can participate.
 ---
 
-# Heading 1 (Page Title)
+## What is a NFT?
+A Non-fungile Token (NFT) was first invented by the Ethereum Community and describes as the ERC721 Token standart. The difference to normal tokens and a NFT is that, the NFT is unique and clearly verifiable and can be used to prove ownership of any digital good.
 
-content - introduction to the topic
+There are many different use cases for NFTs, the most common use case is to link an image to it which becomes unqiue digital art which can be sold on open NFT marketplaces.
 
-## Heading 2 (Sub topic 1)
+Other use cases would be to use NFts as assets in games, for event tickets, for Decentraliced Finance (DeFi), for votings in dezentralced autonomous organisations (DAOs) or for any objects in the Metaverse.
 
-content
+## Different NFT Types on IOTA 
 
-## Heading 2 (Sub topic 2)
+- Layer 1 Assets also known as "Digital Native Assets". With the [IOTA tokenization framework](https://blog.iota.org/tokenization-on-the-tangle-iota-digital-assets-framework/) you can create NFTs on L1 using the NFT output type. Layer 1 NFTs can be transfered on the whole layer without any fees and enable you to transfer NFTs between different smart contract chains.
```

Comment:
Maybe additional h3 headlines for the different NFT types like `### Digital Native Assets` and `### Layer 2 Assets` are more clear than this `ul`

Reply:
Yeah i think that would be nicer too